src/frontend/src/components/landingPage.ts

```diff
@@ -0,0 +1,58 @@
+import { html, TemplateResult } from "lit-html";
+import { navigationLink } from "./footer";
+import { githubIcon, icLogo, questionIcon } from "./icons";
+
+/**
+ * Landing page template
+ *
+ * It is a component with split panes left and right in desktop view.
+ * To the left there is some static content, and to the right there is a slot.
+ *
+ * In movile view, the static content is below the slot.
+ *
+ */
+export const landingPage = ({
+  slot,
+}: {
+  slot: TemplateResult;
+}): TemplateResult => {
+  return html` <main class="c-landingPage">
+    <div class="c-landingPage__logo">
+      <div class="c-logo">${icLogo}</div>
+    </div>
+    <section class="c-landingPage__right">
+      <div class="c-landingPage__right__content">
+        <div class="c-landingPage__right__content--full-width">${slot}</div>
+      </div>
+    </section>
+    <section class="c-landingPage__left">
+      <div class="c-landingPage__left__content">
+        <h1 class="t-title t-title--main">
+          Secure, seamless & privacy-preserving digital identity
```

Comment:
In the other layout there is a special (hidden) title for accessibility. I think for screen readers the landing page should still say "Internet Identity" first (which it presumably does in the old layout due to the hidden title?).

Or is the accessibility concern appropriately addressed by [this](https://github.com/dfinity/internet-identity/pull/2494#issuecomment-2152207910)?

Reply:
It was addressed by https://github.com/dfinity/internet-identity/pull/2494#issuecomment-2152207910, yes.

The logo appears first with the title Internet Identity.